routes/api/comments/index.js

```diff
@@ -1,147 +1,111 @@
 const express = require('express');
 const Comment = require('../../../models/comment');
+const wordlist = require('../../../services/wordlist');
 
 const router = express.Router();
 
-//==============================================================================
-// Get Routes
-//==============================================================================
-
 router.get('/', (req, res, next) => {
-  Comment.find({}).then((comments) => {
-    res.status(200).json(comments);
-  })
-  .catch(next);
-});
-
-router.get('/:comment_id', (req, res, next) => {
-  Comment
-    .findById(req.params.comment_id)
-    .then(comment => {
-      res.status(200).json(comment);
-    })
-    .catch(next);
-});
-
-// Get all the comments that have an action_type over them.
-router.get('/action/:action_type', (req, res, next) => {
-  Comment
-    .findByActionType(req.params.action_type)
-    .then((comments) => {
-      res.status(200).json(comments);
-    })
-    .catch(next);
-});
-
-// Get all the comments that were rejected.
-router.get('/status/rejected', (req, res, next) => {
-  Comment.findByStatus('rejected').then(comments => {
-    res.status(200).json(comments);
+  let query;
+
+  if (req.query.status) {
+    query = Comment.findByStatus(req.query.status);
+  } else if (req.query.action_type) {
+    query = Comment.findByActionType(req.query.action_type);
+  } else {
+    query = Comment.all();
```

Comment:
This endpoint is used by the admin app to retrieve flagged/rejected comments. When we are ready with the FE, middleware will be here to prevent anyone except admin's from hitting the endpoint.